common/navbar.js

```diff
@@ -59,12 +60,16 @@
                 }, function(error) {
                     // No session = no user
                     scope.user = null;
+                    // TODO: Throw the error
```

Comment:
We don't necessarily want to throw an error if the user is not in the system. The user can proceed with some of the functionality not logged in.